### ISSUE TITLE
Search results dt and dd should be contained by dl

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/ppa-django#237]
in the archive search: [<dt> and <dd> elements must be contained by a <dl>](https://dequeuniversity.com/rules/axe/3.1/dlitem?application=AxeChrome) (classified as “critical”)

**Comment by thatbudakguy** [comment]
this is actually a [known aXe issue](https://github.com/dequelabs/axe-core/issues/262); per w3c spec you are allowed to use `<div>` as a container within `dl` for styling purposes, which is exactly what we're doing.
